.github/workflows/test_setup.yml

```
@@ -13,14 +14,14 @@ jobs:
           version: 3.0.3
       - name: Use command
         run: polaris version
-# @rbren this will only work once merged to your repo
-#  build-ext:
-#    runs-on: ubuntu-latest
-#    steps:
-#      - uses: actions/checkout@v2
-#      - name: Setup polaris
-#        uses: fairwindsops/polaris@master
-#        with:
-#          version: 3.0.3
-#      - name: Use command
-#        run: polaris version
+
+  build-ext:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup polaris
+        uses: fairwindsops/polaris@master
```

```suggestion
        uses: fairwindsops/polaris/.github/actions/setup-polaris@master
```
I think it has to be like this.
This afaik is the drawback of having the action in the same repo as the "source".